### PR TITLE
Tech extensions scenario prototype

### PR DESCRIPTION
**Pull request by lvydra**

### CODE REVIEW

File: ui-tests/src/test/resources/features/technical-extension.feature [hunk 1]
```diff
@@ -0,0 +1,73 @@
...
+    And click on the "Technical Extensions" tab
+    Then she is presented with the Syndesis page "Technical extensions"
+    
+  @tech-extension-import-new-tech-extension
+  Scenario: Navigate to technical extensions page
+    When Camilla clicks on the "Import technical extension" button
+    Then she is presented with the Syndesis page "Import Technical Extensions"
+    
+    When Camilla clicks on the "Choose File" button
+    Then she is presented with dialog page "Select a file to upload"
```
Comment (dsimansk) [inline ui-tests/src/test/resources/features/technical-extension.feature]: @lvydra have you managed to locate such a file. Some extension example binary?

**Comment by dsimansk** [comment]
Thanks @lvydra. One suggestion I would like to see one more scenario to cover CRUD operation on Extension management page. 

@lburgazzoli pls could you take a look & review also?

**Comment by lburgazzoli** [comment]
I suggest to add scenario to cover:

- updating a tech extension
- delete a tech ext

**Comment by lvydra** [comment]
@dsimansk Updated

**Comment by lburgazzoli** [comment]
lgtm at this stage, maybe something will need to be refined once we get the final UI

**Pull request merged**